src/convertToObject.js

```diff
@@ -6,7 +6,44 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitArray = sourceString.trim().split(';');
+  const object = {};
+
+  for (const item of splitArray) {
+    if (item.trim().length > 0) {
+      const resultItemSplit = item.split(':');
```

Comment:
The `split` operation could potentially result in more than two items if the CSS declaration contains colons in the value part. You should limit the `split` operation to only split into two parts to avoid losing any part of the value.